src/conv/k-g/arbs.cpp

```diff
@@ -81,19 +81,28 @@ std::map<std::string, rt_arb_internal> Arbs::getArbs(void) const
 }
 
 
-void Arbs::write
+std::vector<std::string> Arbs::write
 (
     rt_wdb* wdbp
 ){
+    std::vector<std::string> ret;
+
     for (std::map<std::string, rt_arb_internal>::iterator it = arbs.begin(); it != arbs.end(); it++) {
+	std::string arbName = name;
+	arbName += it->first;
```

Comment:
Maybe, putting a point "." between the base name and the index?

Reply:
yes it's better , I will update

Reply:
I just have a question about the constructors and destructors, first the arb constructor 
![image](https://github.com/user-attachments/assets/08fdee14-6824-4f2a-94f2-1dbf265b3c5b)
is RT_ARBN_INTERNAL_MAGIC the one to go even with arb4, arb5, arb6, arb7? 
and for Geomtry I  think it doesn't need a constructor or a destructor or am I wrong?

Reply:
then I will handle the other arbs and I can clean the code

Reply:
You need the RT_ARB_INTERNAL_MAGIC here. This is for the rt_arb_internal.

arbn is very different from the arb4 ... arb8.

Reply:
For the Geometry class, the standard constructor and destructor should do.